commands/watch.ts

```diff
@@ -181,11 +206,31 @@ exports.builder = yargs => {
     default: false,
   });
 
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addCmsPublishModeOptions(yargs, { write: true });
-  addUseEnvironmentOptions(yargs);
-  addGlobalOptions(yargs);
-
-  return yargs;
+  return yargs as Argv<WatchCommandArgs>;
+}
+
+const builder = makeYargsBuilder<WatchCommandArgs>(
+  watchBuilder,
+  command,
+  describe,
+  {
+    useConfigOptions: true,
+    useAccountOptions: true,
+    useGlobalOptions: true,
+    useEnvironmentOptions: true,
+    useCmsPublishModeOptions: { write: true },
```

Comment:
I don't think this will work here. This will need to go in the watchBuilder

Reply:
@kemmerle added it in one of her most recent PRs